R/latent_individual.R

```diff
@@ -155,6 +155,11 @@ epidist_formula.epidist_latent_individual <- function(data, family, formula,
   epidist_validate(data)
   formula <- brms:::validate_formula(formula, data = data, family = family)
 
+  # Here could detect which dpar do not have a formula and change it so that
+  # they do. Obviously this isn't a model-specific task so perhaps it's
+  # suggesteing that we want to be generalising epidist_formula in some way
+  # Or it could be a helper function and we put it in all of the epidist_formula
+  
   formula <- stats::update(
```

Comment:
This is the only `latent_individual` specific part of `epidist_formula.latent_individual`.

Create a new internal S3 which does this and make the rest of it a regular function. `as_epidist_formula`, `convert_formula_to_epidist_model`... ask ChatGPT.

Reply:
#372